Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# exit on errors
+set -e
+
+CIRCLE_BRANCH=${CIRCLE_BRANCH:-unknown}
+CIRCLE_BUILD_NUM=${CIRCLE_BUILD_NUM:-00000}
+
+if [[ -z "$GCS_MARKET_KEY" ]]; then
+    echo "$GCS_MARKET_KEY not set aborting!"
+    exit 1
+fi
+
+echo "Preparing content packs for testing ..."
+
+KF=$(mktemp)
+echo "$GCS_MARKET_KEY" > "$KF"
+gcloud auth activate-service-account --key-file="$KF" > auth.out 2>&1
+rm "$KF"
+echo "Auth loaded successfully."
+
+GCS_MARKET_BUCKET="marketplace-dist"
+GCS_BUILD_BUCKET="marketplace-ci-build"
+SOURCE_PATH="content/packs"
+TARGET_PATH="content/builds/$CIRCLE_BRANCH/$CIRCLE_BUILD_NUM"
+echo "Copying master files at: $SOURCE_PATH to target path: $TARGET_PATH ..."
+gsutil -m cp -r "gs://$GCS_MARKET_BUCKET/$SOURCE_PATH" "gs://$GCS_BUILD_BUCKET/$TARGET_PATH"
```

Comment:
this Is a delicate issue from where to copy. we need to check if the uploader can apply all the changes from the branch.